docfx.json

```diff
@@ -205,7 +205,7 @@
         "docs/csharp/roslyn-sdk/tutorials/*.md" : "tutorial",
         "docs/csharp/tour-of-csharp/tutorials/**" : "tutorial",
         "docs/csharp/tutorials/**" : "tutorial",
-        "docs/framework/**/*how-to*.md": "how-to",
+        "docs/framework/**/how-to*.md": "how-to",
```

Comment:
I removed the starting asterisk, because I feel that files like these shouldn't be labeled as how-to topics.

![image](https://user-images.githubusercontent.com/24882762/116462234-133a3880-a81e-11eb-98f8-527537775d1c.png)